audit/projects/k8s-infra-prow-build/secrets/prow-build-ssh-key-secret-ssh-public/versions.json

```diff
@@ -1,4 +1,13 @@
 [
+  {
+    "createTime": "2021-11-12T00:20:15.839963Z",
+    "etag": "\"15d08c6c60dedb\"",
+    "name": "projects/773781448124/secrets/prow-build-ssh-key-secret-ssh-public/versions/4",
```

Comment:
Audit logs suggest it's @cjwagner : 

```
 Secret Manager API AddSecretVersion prow-build-ssh-key-secret-ssh-public:4 colew@google.com
 ```

Reply:
Yes, this was part of the SSH key rotation.